packages/quantic/force-app/main/default/lwc/quanticNumericFacet/quanticNumericFacet.html

```diff
@@ -25,7 +25,7 @@
           <ul class="slds-has-dividers_around-space slds-var-m-top_medium">
             <template if:true={withInput}>
               <div class="slds-p-horizontal_x-small">
-                <form onsubmit={onApply} class="facet__search-form slds-grid slds-gutters slds-var-p-bottom_small slds-var-p-top_small">
+                <form onsubmit={onApply} class="slds-grid slds-gutters slds-var-p-bottom_small slds-var-p-top_small">
```

Comment:
I need this `facet__search-form` for cypress test

Reply:
i must've made a mistake with my merged after reopening this branch. good catch!